tests/examples_methods_syntax/grouped_bar_chart_overlapping_bars.py

```diff
@@ -0,0 +1,27 @@
+"""
+Grouped Bar Chart with xOffset and overlapping bars
+---------------------------------------------------
+Like :ref:`gallery_grouped_bar_chart2`, this example shows a grouped bar chart using the ``xOffset`` encoding channel, but in this example the bars are partly overlapping within each group.
+"""
+# category: bar charts
+import altair as alt
+import pandas as pd
+
+source = pd.DataFrame(
+    {
+        "category": list("AABBCC"),
+        "group": list("xyxyxy"),
+        "value": [0.1, 0.6, 0.7, 0.2, 0.6, 0.1],
+    }
+)
+
+base = alt.Chart(source, width=alt.Step(12)).encode(
+    x="category:N",
+    y="value:Q",
+    xOffset=alt.XOffset("group:N").scale(paddingOuter=0.5),
+)
+
+alt.layer(
+    base.mark_bar(size=20, stroke="white", fillOpacity=0.9).encode(fill="group:N"),
+    base.mark_text(dy=-5).encode(text="value:Q"),
```

Comment:
```suggestion
    base.mark_text(dy=10, color="white").encode(text="value:Q"),
```

You may have tried this already, but wanted to add for comparison.

I think this makes the lower value *easier* to read for 2/3 of the lower values.

``{"category": "A", "group": "x"}`` still isn't great though due to the interaction with the bar stroke.

<img height="340" alt="original" src="https://github.com/user-attachments/assets/edf24fbb-5b7c-46f0-a16c-ca4709107684"> <img height="340" alt="alternative" src="https://github.com/user-attachments/assets/7daf604d-d0ab-460a-b938-c816c818919c">